man/rules/meson.build

```diff
@@ -128,6 +128,9 @@ manpages = [
    'SD_ID128_NULL',
    'SD_ID128_UUID_FORMAT_STR',
    'sd_id128_equal',
+   'sd_id128_equals',
```

Comment:
btw, it's a bit weird that "sd_id128_equal" can deal with one parameter, and "sd_id128_equals" with many. If you read "equals" as verb, then the suffix "s" kinda suggests a singular subject. I mean, it's "the value equals 5" but the "the freedoms equal burdens", i.e. the first is singular and the second is plural but the "s" on the function is used the opposite way.

Many call this sd_id128_in_set()?

Reply:
+1 for _in_set

Reply:
I changed it to sd_id128_in_set, sd_id128_in_setv, sd_id128_in_set_sentinel.